Resources/Prototypes/Traits/skills.yml

```diff
@@ -168,3 +168,19 @@
   components:
     - type: Singer
       proto: NormalSinger
+
+- type: trait
```

Comment:
Are the other traits not coming in this PR?

Reply:
The other two traits are not coming in this PR, and will be a separate PR. One PR that refactors Psionic Insulation to make it event based, with a single central location for tracking Insulation effects, which will include the x-Waveform Misalignment trait. And a second PR that refactors Telepathy, and re-introduces the split between Latent Psychic(Having a Blank Psionic Component), and Natural Telepath(Which will be Psionic Component, with roundstart TelepathyPower). I intend to in a separate PR from this, make Telepathy an ActivePower that you have to roll for.